ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -254,6 +256,49 @@ impl Query {
         .await
     }
 
+    async fn github_repository_provider(
```

Comment:
instead of adding a new method, just add `ids: Option<Vec<ID>>` to `github_repository_providers`?